src/inversion_scripts/run_inversion.sh

```diff
@@ -103,14 +103,14 @@ printf "DONE -- postproc_diags.py\n\n"
 #=======================================================================
 
 if ! "$PrecomputedJacobian"; then
-
-    # 50% perturbation
-    Perturbation=0.5
-
+    python_args=(calc_sensi.py $nElements $PerturbValue $StartDate $EndDate $JacobianRunsDir $RunName $sensiCache)
+    # add an argument to calc_sensi.py if optimizing BCs
+    if "$OptimizeBCs"; then
+        python_args+=($PerturbValueBCs)
+    fi
```

Comment:
This is cool. Just checking, does it add the necessary space between the last argument `$sensiCache` and the new one `$PerturbValuesBCs` ?

Reply:
Same question below.

Reply:
Yes, the reason being that `python_args` is now an array, so this actually just appends `$PertubValueBCs` to the array, then it gets unpacked for the actual python call (spaces automatically added).